scripts/setup_colab.sh

```diff
@@ -116,7 +116,7 @@ if [[ -z "`command -v file`" ]]; then
   apt-get install -y -q file libmagic-dev
```

Comment:
please try this script on colab and verify it works before submitting

Reply:
Tested on colab and fixed. Works now.
https://colab.research.google.com/drive/1O_hDHDJqxkp2L1Lw4sHknKGFRgne20nE